scss/_variables.scss

```diff
@@ -260,6 +261,20 @@ $transition-fade:             opacity .15s linear !default;
 $transition-collapse:         height .35s ease !default;
 
 
+// Responsive font sizes
+//
+// RFS Configuration.
+
+$rfs-minimum-font-size: 1rem !default;
+$rfs-font-size-unit:    rem !default;
+$rfs-breakpoint:        1200px !default;
+$rfs-factor:            5 !default;
+$rfs-two-dimensional:   false !default;
+$rfs-class:             false !default;
+
+// This variable generates the `.rfs-*` classes.
+$responsive-font-sizes: () !default;
```

Comment:
No, I've always only added the font sizes I needed here. Not sure if we should defaults here?

Reply:
I've removed the `$responsive-font-sizes` variable. I think it's better to look how we're going to approach the font size utilities in v5 instead of providing an empty sass map for now.